src/DurableTask.AzureStorage/Partitioning/TablePartitionManager.cs

```
@@ -584,29 +585,28 @@ public async Task BalanceLeases(Dictionary<string, List<TableLease>> partitionDi
                         catch (RequestFailedException ex) when (ex.Status == 412)
                         {
                             this.settings.Logger.PartitionManagerInfo(
-                            this.storageAccountName,
-                            this.settings.TaskHubName,
-                            this.settings.WorkerId,
-                            partition.RowKey,
-                            $"Failed to update {partition.RowKey} due to concurrency. Re-read the table to get the latest Etag.");
+                                this.storageAccountName,
+                                this.settings.TaskHubName,
+                                this.settings.WorkerId,
+                                partition.RowKey,
+                                $"Failed to update table entry due to an Etag mismatch. Failed ETag value: '{etag}'");
                             throw;
                         }
                         // Eat any exceptions during lease stealing.
                         catch (Exception exception)
                         {
-                            this.settings.Logger.PartitionManagerError(
+                            this.settings.Logger.PartitionManagerWarning(
```

I don't feel strongly about this, but I usually try to use Error only when something really bad actually happens. That's usually not the case when we swallow exceptions like this.